app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Formatters.scala

```diff
@@ -24,21 +24,28 @@ import scala.util.control.Exception.nonFatalCatch
 
 trait Formatters {
 
+  private lazy val twoDecimalPattern = """(\d*\.[0-9]{1,2})""".r
+  private lazy val decimalRegexp     = """(\d*\.\d*)""".r
+
+  private def removeWhitespace(value: String, removeAllWhitespace: Boolean) =
+    if (removeAllWhitespace) value.filterNot(_.isWhitespace) else value.strip()
+
+  private def removePoundSign(value: String) =
+    value.filterNot(_ == '£')
```

Comment:
This will remove the £ wherever it is. You only need to remove if it is the first character. Input is invakid if the £ sign is anywhere else.

Reply:
Will update it, good point